packages/ui-components/src/components/date-time-input/date-time-input.tsx

```diff
@@ -150,6 +155,7 @@ export class DateTimeInput implements IDateTimeInput, IDateTimeInputEvents {
 								ref={input => (this.nativeInput = input as HTMLInputElement)}
 								type={EInputFieldType.Text}
 								name={this.inputName}
+								disabled={this.disabled}
```

Comment:
Should the input be focusable if it's disabled?

Reply:
Changed bellow to prevent this to be focusable if disabled